consul/src/main/scala/io/buoyant/consul/CatalogNamer.scala

```diff
@@ -51,11 +51,11 @@ class CatalogNamer(
     def get(name: String): DcCache = synchronized {
       activity.sample() match {
         case Activity.Ok(snap) =>
-          snap.get(name).getOrElse {
+          snap.getOrElse(name, {
```

Comment:
Thanks for pointing me to that discussion.  Option's `getOrElse` also takes a closure:
```def getOrElse[B >: A](default: => B): B```
Am I missing something?